deploy/development.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
 kubectl delete deployments.apps hacbs-jvm-operator -n jvm-build-service
-kubectl delete deployments.apps hacbs-jvm-cache -n jvm-build-service
-kubectl delete deployments.apps localstack -n jvm-build-service
+#kubectl delete deployments.apps hacbs-jvm-cache -n jvm-build-service
```

Comment:
can you clarify via comment why we do not want to recycle these, so in a few weeks if I'm back in this script, I don't inadvertently add these deletes back

Reply:
Done. Basically this is only really necessary if you are actively working on the cache, otherwise it just slows down the development flow as builds end up taking longer.